docs/en/observability/monitor-infra/kubernetes-pod-metrics.asciidoc

```diff
@@ -0,0 +1,28 @@
+[[kubernetes-pod-metrics]]
+= Kubernetes pod metrics
+
+//TODO: Rewrite this to match new format. Requires input from dev team.
+
+//REVIEWERS: This was taken from the existing docs and moved here. Do we want
+//to publish this info for now until updated desriptions are available?
+
+Learn about Kubernetes pod metrics displayed in the {infrastructure-app}.
+
+|=== 
+
+| *CPU Usage* | Average of `kubernetes.pod.cpu.usage.node.pct`. 
+
+| *Memory Usage* | Average of `kubernetes.pod.memory.usage.node.pct`.
+
+| *Inbound Traffic* | Derivative of the maximum of `kubernetes.pod.network.rx.bytes` scaled to a 1 second rate.
+
+| *Outbound Traffic* | Derivative of the maximum of `kubernetes.pod.network.tx.bytes` scaled to a 1 second rate.
+
+|=== 
+
+//REVIEWERS: Is the topic we are pointing to here accurate? It seems like it
```

Comment:
Reviewers: Please respond to this question about the Infrastructure app field reference. If that topic needs updates, we'll need to do it in a follow-on PR. The main question now is whether we should like to it.

Reply:
Hey @dedemorton - yes, that's still accurate. OK to publish